app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -2612,7 +2614,12 @@ private void updateCurrentBgInfoCommon(DexCollectionType collector, TextView not
         }
 
         if (!isSensorActive) {
-            notificationText.setText(R.string.now_start_your_sensor);
+            boolean G6_not_connected_yet = Pref.getBooleanDefaultFalse("using_g6") && DexTimeKeeper.getTransmitterAgeInDays(getTransmitterID()) == -1;
```

Comment:
You could stick to the Java convention that variable names should be camelCase, i.e. start with a lower character.
```suggestion
            boolean g6NotConnectedYet = Pref.getBooleanDefaultFalse("using_g6") && DexTimeKeeper.getTransmitterAgeInDays(getTransmitterID()) == -1;
```

Reply:
My objective is to get this change added so that xDrip functionality improves for the inexperienced user.
I am more than happy to make any necessary change to get approval.

Therefore, if I am told that this is what's holding this PR up, I will change it.

Let me add that looking at this same file, I can see plenty of existing uppercase instances.  For example: https://github.com/NightscoutFoundation/xDrip/blob/35d548f08bee8289a3a18e9d825b3c8145135921/app/src/main/java/com/eveningoutpost/dexdrip/Home.java#L258

So, SHOWCASE_G5FIRMWARE is already there.  If we are going to change the code for some reason, we should be consistent.
If, we want to say, let's leave the existing cases alone.  But, let's follow this guideline for the new variables, I am happy to follow such a guideline if I can get an approval of it.  

So, I thank you for having a look and making the suggestion.  But, I am not going to make your requested change because I have no idea it is going to make any difference in what really matters in getting this approved or not.

Reply:
It just caught my eye when skimming through your PR. The constant in line 258 is fine by convention. https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
Thanks a lot for providing this guidance.
I will change to follow the standard.  I have no reason not to follow it.